File: frontend/src/app/dto/order/order-request.ts
export interface OrderRequest {
    userId: number;
    recipientName: string;
    contactEmail: string;
    shippingAddress: string;
    contactPhone: string;
    items: OrderItemRequest[];
  }
  
  export interface OrderItemRequest {
    menuItemId: number;
    quantity: number;
  }

  

File: order-service/src/main/java/com/vanhuy/order_service/dto/OrderRequest.java
package com.vanhuy.order_service.dto;

import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@Builder
public class OrderRequest {
    private Integer userId;
    private String recipientName;
    private String contactEmail;
    private String shippingAddress;
    private String contactPhone;
    private List<OrderItemRequest> items;
}
